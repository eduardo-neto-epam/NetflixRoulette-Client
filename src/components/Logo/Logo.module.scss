.brand {
        font-family: var(--font-main);
        color:  var(--color-primary);
        font-size: 20px;
        font-weight: var(--weight-regular);

    &--bold {   
        font-weight: var(--weight-bold);
    }
}