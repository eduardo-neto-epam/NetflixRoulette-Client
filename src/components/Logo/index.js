import React, { memo } from 'react'
import styles from './Logo.module.scss'
import classnames from 'classnames/bind'

let cx = classnames.bind(styles)

const Logo = () => {
    return (
        <div>
            <span className={cx('brand', 'brand--bold')}>netflix</span>
            <span className={cx('brand')}>roulette</span>
        </div>
    )
}

export default memo(Logo);
