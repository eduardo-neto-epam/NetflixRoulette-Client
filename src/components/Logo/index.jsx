import React, { memo } from 'react';
import classnames from 'classnames/bind';
import styles from './Logo.module.scss';

const cx = classnames.bind(styles);

const Logo = () => (
  <div>
    <span className={cx('brand', 'brand--bold')}>netflix</span>
    <span className={cx('brand')}>roulette</span>
  </div>
);

export default memo(Logo);
