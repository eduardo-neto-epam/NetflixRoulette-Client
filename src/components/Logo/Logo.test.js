import renderer from 'react-test-renderer';
import React from 'react'
import Logo from './index'

test('Logo renders correctly', () => {
    const tree = renderer
        .create(<Logo />)
        .toJSON();
    expect(tree).toMatchSnapshot();
  });