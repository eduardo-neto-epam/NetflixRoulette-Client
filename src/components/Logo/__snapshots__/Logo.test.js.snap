// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<div>
  <span
    className="brand brand--bold"
  >
    netflix
  </span>
  <span
    className="brand"
  >
    roulette
  </span>
</div>
`;
